src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressNode.cs

```diff
@@ -56,7 +64,16 @@ namespace Microsoft.PowerShell
             this.PercentComplete = Math.Min(record.PercentComplete, 100);
             this.SecondsRemaining = record.SecondsRemaining;
             this.RecordType = record.RecordType;
-            this.Style = RenderStyle.FullPlus;
+
+            if (ExperimentalFeature.IsEnabled("PSAnsiProgress") && PSStyle.Instance.Progress.View == ProgressView.Minimal)
```

Comment:
Rather than writing this double-barrelled condition out everywhere, it might be DRYer to write a Boolean method (the JIT would probably favour it too):

```csharp
internal static bool IsMinimalProgressRenderingEnabled()
{
    return ExperimentalFeature.IsEnabled("PSAnsiProgress")
        && PSStyle.Instance.Progress.View == ProgressView.Minimal;
}
```